src/main/java/org/opensearch/performanceanalyzer/http_action/config/PerformanceAnalyzerOverridesClusterConfigAction.java

```diff
@@ -15,9 +15,9 @@
 import org.apache.logging.log4j.Logger;
 import org.opensearch.client.node.NodeClient;
 import org.opensearch.common.settings.Settings;
-import org.opensearch.common.xcontent.XContentBuilder;
 import org.opensearch.common.xcontent.XContentHelper;
 import org.opensearch.common.xcontent.XContentType;
+import org.opensearch.core.xcontent.XContentBuilder;
```

Comment:
Only XContentBuilder moved to `core` and not the others in `common`?

Reply:
Based on the commit in core, the current initiative is to only move the xcontent base classes to common, see more detail at https://github.com/opensearch-project/OpenSearch/commit/cb402cb867d05dc6e33a7b19f8d95548cfb3d445